src/components/alert/alert.js

```diff
@@ -24,7 +24,12 @@ import Dialog from '../dialog';
  */
 class Alert extends Dialog {
 
+  constructor(props) {
+    super(props);
```

Comment:
Is adding this constructor necessary? There doesn't appear to be any new functionality added.